ui/src/main/java/org/pentaho/di/ui/spoon/DatabasesCollector.java

```diff
@@ -40,36 +39,23 @@
 public class DatabasesCollector {
   private DatabaseManagementInterface dbManager;
   private final Repository repository;
-  private AbstractMeta meta;
   private List<String> dbNames;
   private Map<String, DatabaseMeta> dbMetaMap;
 
   public DatabasesCollector( DatabaseManagementInterface dbManager, Repository repository ) {
-    this( dbManager, null, repository );
-  }
-
-  public DatabasesCollector( DatabaseManagementInterface dbManager, AbstractMeta meta, Repository repository ) {
     this.dbManager = dbManager;
-    this.meta = meta;
     this.repository = repository;
   }
 
   public void collectDatabases() throws KettleException {
     dbMetaMap = new HashMap<String, DatabaseMeta>();
     List<DatabaseMeta> dbMetaList;
 
-    // SharedObjects
     if ( dbManager != null ) {
       dbMetaList = dbManager.getDatabases();
       addToMetaMap( dbMetaList );
     }
 
-    // local databases
```

Comment:
This is going to break how we display the local db connections in UI

Reply:
See the calling code. Now AbstractMeta *also* uses DatabaseManagementInterface, so there isn't a reason to use both modes.